alembic/operations/ops.py

```diff
@@ -1094,7 +1097,10 @@ class DropTableOp(MigrateOperation):
     def __init__(self, table_name, schema=None, table_kw=None, _reverse=None):
```

Comment:
```suggestion
    def __init__(self, table_name, schema=None, table_kw=None, _reverse=None, **kw):
        if table_kw:
            warnings.warn('...', DeprecationWarning)
            kw.update(table_kw)
        self.table_name = table_name
        self.schema = schema
        self.comment = kw.pop('"comment", None)
        self.info = kw.pop('"info", None)
        self.prefixes = kw.pop("prefixes", None)
        self.table_kw = kw
```
Would something like it help transition ? I don't like it much, because it would allow to pass twice the same "kwargs", with different values, once through `table_kw`, once though the regular `**kw`.
I would at least add a check that there is no override. It would also requite to fix the `drop_table()` class method.

Reply:
i wouldnt want them both at the same time for the moment.    again I want to get the feature "working" and then if we want to consolidate/clean up/use both/one or the other of "kw" / "table_kw" i want to look at that separately.